src/ceph-volume/plugin/zfs/README.rst

```diff
@@ -0,0 +1,24 @@
+==================
```

Comment:
might want to remove this file. as most of its content is not relevant. and the link in it is not reachable.